### ISSUE TITLE
Error in progress

### ISSUE THREAD

**Issue by rinne13** [bleskodev/rubyripper#611]
I got error this error when using it in arch linux:

/usr/lib/rubyripper/gtk2/ripStatus.rb:40:in `%': incomplete format specifier; use %% (double %) instead (ArgumentError)

Could be solved with the lines modified in ripStatus.rb:
```
  Show the update in progress of the ripping/encoding
  def updateProgress(type, value)
    progress = "%.3g" % [ (value * 100) , '%' ]
    if type == 'encoding'
      @encBar.text = _("Encoding progress %s \%%") % [progress]
      @encBar.fraction = value
    else
      @ripBar.text = _("Ripping progress %s \%%") % progress
      @ripBar.fraction = value
    end
  end
```

**Comment by Warepire** [comment]
I think I fixed that one, please try the PR I opened #613

**Comment by bleskodev** [comment]
Should be fixed with the mentioned PR (which has been merged).
Thanks for the fix!
